FABulous/fabric_generator/fabric_gen.py

```diff
@@ -420,6 +420,7 @@ def generateConfigMem(self, tile: Tile, configMemCsv: str) -> None:
             self.writer.addPreprocElse()
         self.writer.addNewLine()
         self.writer.addNewLine()
+        self.writer.addLogicStart()
```

Comment:
I have started working on the new code generator since I need it to generate techmap map automatically. The new generator can do 
```python
with codeRegion():
   statement()
   statement()
```
Which then can avoid problem like this.

Reply:
Sounds cool! 
Looking forward to this, since the current code generation can really cause a lot of issues.